beekeeper-integration-tests/src/test/resources/message.json

```diff
@@ -7,5 +7,5 @@
   "Signature": "signature",
   "SigningCertURL": "https://sns.us-west-2.amazonaws.com/SimpleNotificationService-xxxx",
   "UnsubscribeURL": "https://sns.us-west-2.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-west-2:440407435941:sns-topic",
-  "Message": "{ \"protocolVersion\": \"1.0\", \"eventType\": \"DROP_TABLE\", \"dbName\": \"some_db\", \"tableName\": \"some_table\", \"tableLocation\": \"%s\", \"tableParameters\": { \"beekeeper.unreferenced.data.retention.period\": \"P7D\", \"beekeeper.remove.unreferenced.data\": \"%s\",  \"beekeeper.hive.event.whitelist\":\"%s\" }}"
+  "Message": "replace"
```

Comment:
Up to you, but it be clearer for the string to replace to be something like `${replace}` as that's a convention used in quite a few templating libraries which is the intention here.